src/components/Image/types.ts

```diff
@@ -0,0 +1,45 @@
+import {ImageRequireSource, ImageResizeMode, ImageStyle, ImageURISource, StyleProp} from 'react-native';
+import {ImageStyle as FastImageStyle, OnLoadEvent, ResizeMode, Source} from 'react-native-fast-image';
+import {OnyxEntry} from 'react-native-onyx';
+import {Session} from '@src/types/onyx';
+
+type ImageOnyxProps = {
+    /* Onyx Props */
```

Comment:
Comment unnecessary, the name of type speaks for itself.

```suggestion
```

Reply:
Done